src/Turbo/composer.json

```diff
@@ -42,7 +42,7 @@
     "require-dev": {
         "doctrine/annotations": "^1.12",
         "doctrine/doctrine-bundle": "^2.2",
-        "doctrine/orm": "^2.8",
+        "doctrine/orm": "~2.8.0",
```

Comment:
I know this gets the tests passing, but let's revert this as it's unrelated.

Reply:
Alright

Reply:
Moved to https://github.com/symfony/ux/pull/92